Dockerfile

```diff
@@ -1,14 +1,20 @@
 # syntax=docker/dockerfile:experimental
 FROM python:3-alpine
 
+ARG BUILD_DIR=/build
+
 # Add img
 RUN apk add img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 
 # Setup other deps
 RUN apk add git skopeo
 
 # Install taskboot from mounted source code
-RUN --mount=type=bind,target=/src/taskboot,readwrite \
-  cd /src/taskboot && python setup.py install
+# Use a dedicated build dir to avoid building in bind mount
+RUN --mount=type=bind,target=/src/taskboot \
+  cd /src/taskboot && \
+  mkdir -p ${BUILD_DIR} && \
+  pip install --build ${BUILD_DIR} . && \
```

Comment:
Do you want to use --no-cache-dir to reduce the size?

Reply:
It removes a whopping 3MB, more than 2 :floppy_disk: !

```
ls -alh x.tar y.tar 
-rw-r--r-- 1 lao lao 178M mai   13 10:30 x.tar
-rw-r--r-- 1 lao lao 175M mai   13 10:29 y.tar
```

Reply:
Wow, huge savings!